### PR TITLE
Add tests for dnf check feature

### PR DESCRIPTION
**Pull request by janblazek**
This pull request is adding tests for dnf check commands:
--dependencies
--duplicates
--obsoleted

https://taiga.fedorainfracloud.org/project/jsilhan-dnf/task/69

### CODE REVIEW

**Comment by ignatenkobrain** [comment]
Looks good, but should not we test output as well?

**Comment by janblazek** [comment]
You're right. I'll add the checks for output.

File: dnf-docker-test/features/check-1.feature [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      Given repository "base" with packages
+         | Package      | Tag       | Value        |
+         | TestA        | Version   | 1            |
+       When I save rpmdb
+        And I enable repository "base"
+        And I successfully run "dnf -y install TestA"
+       Then rpmdb changes are
+         | State     | Packages       |
+         | installed | TestA          |
+       When I successfully run "dnf check"
```
Comment (ignatenkobrain) [inline dnf-docker-test/features/check-1.feature]: should we check if stdout is empty here?

Comment (janblazek) [inline dnf-docker-test/features/check-1.feature]: Yes, I do agree.

Comment (janblazek) [inline dnf-docker-test/features/check-1.feature]: I have added same checks to check-2.feature and check-3.feature.

**Pull request merged**